src/coreclr/tools/Common/Internal/NativeFormat/NativeFormatWriter.cs

```diff
@@ -2044,12 +2052,20 @@ internal override void Save(NativeWriter writer)
 
         public override bool Equals(object obj)
         {
-            throw new NotImplementedException();
+            // TODO: Support value equality instead of just referential equality
+            return ReferenceEquals(this, obj);
         }
 
         public override int GetHashCode()
         {
-            throw new NotImplementedException();
+            int hashCode = 13;
```

Comment:
What purpose is this hashcode used for?

Reply:
This hash code is needed because we try to insert the `TupleVertex` that contains this hash table instance into the "unified nodes" dictionary.

Reply:
Does the hash code need to be deterministic for that? Can we just use `RuntimeHelpers.GetHashCode(object)`?